module-3/backend/week_one.md

```diff
@@ -5,7 +5,22 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. What is `json` and why is it important?
+```
+JSON is what APIs are generally displayed as (or XML). It takes the information from a database and outputs it so that it can be consumed by a front end or someone else who wants the data. The controller is where json is rendered. 
+```
 2. What's the difference between `joins` and `includes` in ActiveRecord?
+``` 
```

Comment:
Includes works very much like joins, but also pre-loads some of the data on the other table to prevent multiple queries where possible.